pywinauto/unittests/test_uia_element_info.py

```diff
@@ -62,6 +62,34 @@ def testChildren(self):
             """Test whether a list of only immediate children of the element is equal"""
             self.assertEqual(len(self.ctrl.children()), 5)
 
+        def testDefaultDepthDescendants(self):
```

Comment:
Please use PEP-8 style for new test cases. Example: `test_default_depth_descendants`